common/app/views/fragments/page/head/orielScriptTag.scala.html

```diff
@@ -8,7 +8,7 @@
 
 @if(
     ActiveExperiments.isParticipating(OrielParticipation) ||
-            (orielSonobiIntegration.isSwitchedOn && request.uri == "/environment/2018/mar/11/labrador-switch-energy-suppliers")
+            (orielSonobiIntegration.isSwitchedOn && request.path == "/environment/2018/mar/11/labrador-switch-energy-suppliers")
```

Comment:
What's the difference? Does uri include querystring?

Reply:
```
 def uri : String

The complete request URI, containing both path and query string. 
```